includes/admin/class-wc-admin-menus.php

```diff
@@ -184,8 +184,8 @@ public function menu_order_count() {
 
 			// Add count if user has access.
 			if ( apply_filters( 'woocommerce_include_processing_order_count_in_menu', true ) && current_user_can( 'manage_woocommerce' ) ) {
-				$order_count = wc_processing_order_count();
-
+				$order_count = apply_filters( 'woocommerce_modify_processing_order_count_in_menu', wc_processing_order_count() );
```

Comment:
This is not a good place for doing it, if you want to manipulate the total of processing orders you should want change it in `wc_processing_order_count()` or in `wc_orders_count()`, probably better include the filter in the last function, so you can filter any of the results.

Reply:
thanks for the recommendation. I moved it now.

Reply:
Sorry, I'm checking it again, I think will be better where you first included.
This function it's used only in one place and maybe it will get deprecated soon, since there's already the `wc_orders_count()` method.